### PR TITLE
feat: Camel-k 2.0 Micrometer Metrics

### PR DESCRIPTION
**Pull request by gansheer**
Blog post about the change from Microprofile to Micrometrics in Camel-K 2.0

cc @squakez @claudio4j

### CODE REVIEW

File: content/blog/2023/08/camel-k-micrometer/index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+categories: ["Camel K", "Features", "Devops"]
+preview: "Camel K 2.0: what changes to expect about Integration Metrics"
+---
+
+
+We already explained how to take monitor your Integrations in the previous blog post about [monitoring](/blog/2022/07/camel-k-monitoring-ops/) operations on Camel K. The good news is there are only a few changes with the move to [Micrometer Metrics](https://micrometer.io/).
+
+## What is new
+
+Camel-K 2.0 was the occasion to move from one technology (Microprofile) to another (Micrometer) for the [Prometheus trait configuration](/camel-k/next/traits/prometheus.html) implementation.
```
Comment (squakez) [inline content/blog/2023/08/camel-k-micrometer/index.md:15]:
```suggestion
Camel K 2.0 was the occasion to move from one technology (Microprofile) to another (Micrometer) for the [Prometheus trait configuration](/camel-k/next/traits/prometheus.html) implementation.
```

File: content/blog/2023/08/camel-k-micrometer/index.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+draft: false
+authors: [gansheer]
+categories: ["Camel K", "Features", "Devops"]
+preview: "Camel K 2.0: what changes to expect about Integration Metrics"
+---
+
+
+We already explained how to take monitor your Integrations in the previous blog post about [monitoring](/blog/2022/07/camel-k-monitoring-ops/) operations on Camel K. The good news is there are only a few changes with the move to [Micrometer Metrics](https://micrometer.io/).
+
+## What is new
```
Comment (squakez) [inline content/blog/2023/08/camel-k-micrometer/index.md:13]:
```suggestion
## From Microprofile to Micrometer
```

File: content/blog/2023/08/camel-k-micrometer/index.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+Camel-K 2.0 was the occasion to move from one technology (Microprofile) to another (Micrometer) for the [Prometheus trait configuration](/camel-k/next/traits/prometheus.html) implementation.
+
+The reason is the deprecation notice from [Quarkus Microprile's implementation](https://quarkus.io/guides/smallrye-metrics) in favor of using [Micrometer Metrics](https://quarkus.io/guides/telemetry-micrometer). This decision follows a [long period of transition](https://quarkus.io/blog/micrometer-metrics/) from Quarkus project.
+
+This change brings some powerful improvments for your monitoring. Micrometer delivers more advanced features such as hierarchical meters and dimensional data models, while MicroProfile Metrics provides a simpler API with basic metric types such as counters and gauges. Additionaly, Micrometer provides a simple facade over the instrumentation clients for the most popular monitoring systems, letting any library provides its own metrics through it.
+
+## What changes
```
Comment (squakez) [inline content/blog/2023/08/camel-k-micrometer/index.md:21]:
```suggestion
## Camel framework changes expected in Micrometer
```

**Review by squakez — APPROVED** [review]
Some changes to help SEO

**Comment by gansheer** [comment]
> Some changes to help SEO

I didn't even think about SEO, thanks for the tips.

**Review by claudio4j — APPROVED** [review]

**Pull request merged**